src/module-elasticsuite-core/Cluster/ClusterInfo.php

```diff
@@ -56,4 +56,16 @@ public function getServerVersion()
 
         return $this->serverVersion;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getServerDistribution()
+    {
+        if ($this->serverDistribution === null) {
```

Comment:
Can you declare this property ? Build is failing with PHP 7.4 and I think that's because it's less permissive with undeclared variable. For now we still hope our next minor would be compatible will also Magento 2.4.3.

Reply:
Notice: Undefined property: Smile\ElasticsuiteCore\Cluster\ClusterInfo::$se  
  rverDistribution in /var/www/magento/vendor/smile/elasticsuite/src/module-e  
  lasticsuite-core/Cluster/ClusterInfo.php on line 65

Reply:
Eesh sorry about that, I had it in my main test env but forgot for this PR, added now.